### ISSUE TITLE
SQLSTATE[42S02]: Base table or view not found: 1146 Table 'accessibility.pages' doesn't exist (Conne...

### ISSUE THREAD

**Issue by jobara** [accessibility-in-action/platform#1996]
**`Illuminate\Database\QueryException`**

SQLSTATE[42S02]: Base table or view not found: 1146 Table 'accessibility.pages' doesn't exist (Connection: mysql, SQL: select * from `pages` where json_unquote(json_extract(`title`, '$."en"')) = Privacy Policy limit 1)

:dart: Illuminate\Database\Connection::runQueryCallback
:page_facing_up: /vendor/laravel/framework/src/Illuminate/Database/Connection.php

```php
799   }
800   
801*  throw new QueryException(
802       $this->getName(), $query, $this->prepareBindings($bindings), $e
803   );
```
[Open in Flare](https://flareapp.io/errors/4667199/latest)

**Comment by jobara** [comment]
I'm not sure exactly what happened here, but it's like the Global Deploy didn't run. I also noticed that the fonts were missing when you'd visit accessibilityexchange.ca . I manually ran the migrations (I had to run the production seeder this first time anyways) and then also `php artisan google-font:fetch`. After this the fonts worked, and I think the site is working too. But we should figure out what happened with the global deploy.
